README.md

```diff
@@ -1,65 +1,50 @@
 # Helm Hub
 
-The Helm Hub repository contains the documentation and some configuration for the
-distributed public repository search hosted by Helm that can be found at https://hub.helm.sh
+The Helm Hub repository contained the documentation and some configuration for the
+distributed public repository search hosted by Helm that used to be found at https://hub.helm.sh
 
-## Distributed Search
+## NOTICE: Redirect To Artifact Hub
 
-Helm was designed with many distributed repositories in mind. This is why the Helm
-CLI has commands to manage adding, updating, listing, and removing them. Repositories
-are hosted by many people and organizations. Sometimes these are private and sometimes
-they are public with a goal of distributing applications.
+The [Artifact Hub](https://artifacthub.io/) is a CNCF project to make discovering
+distributed cloud native artifacts easy. This includes Helm charts. The Helm Hub,
+including the search in the Helm Client, now redirect to the Artifact Hub. It
+provides a better experience to that of the Helm Hub.
 
-Discovering the charts hosted by the Helm project via the [stable and incubator](https://github.com/helm/charts)
-repositories has been relatively easy. We want to go further and make it possible
-to discover the charts and repositories being hosted by others.
+## Claiming Your Repo In The Artifact Hub
 
-The Hub provides a means to more easily find charts that are hosted outside the Helm
-project. This has implications for those who build and distribute charts including:
+If you had a repository in the Helm Hub but had not listed it in the Artifact
+Hub, it was automatically listed. Everything that as in the Helm Hub should be
+discoverable in the Artifact Hub.
 
-- They can manage the workflow around releases themselves
-- They can track metrics around the use of their charts
+There are two ways to claim your listing so that you can manage it on the Artifact
+Hub:
 
-## Artifact Hub Migration
+1. If you create an account on the Artifact Hub there is a method to claim your
```

Comment:
I seem to remember a link to artifacthub/hub documentation on this. If I'm remembering that right, maybe add a link to that here?

Reply:
Done